BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -190,3 +192,44 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
     }
     
 }
+
+extension StoresTableViewController {
```

Comment:
I'm thinking that we might need to have some form of refresh feature for the mapView? Another topic we need to discuss with Courtney?